Mathlib/MeasureTheory/Constructions/Pi.lean

```diff
@@ -970,6 +970,23 @@ theorem volume_preserving_piFinsetUnion (α : ι → Type*) [DecidableEq ι] {s
     MeasurePreserving (MeasurableEquiv.piFinsetUnion α h) volume volume :=
   measurePreserving_piFinsetUnion h (fun _ ↦ volume)
 
+theorem measurePreserving_pi {β : ι → Type*} [∀ i, MeasurableSpace (β i)]
+    (ν : (i : ι) → Measure (β i)) [∀ i, SigmaFinite (ν i)] {f : (i : ι) → (α i) ≃ᵐ (β i)}
```

Comment:
You are assuming that the `f i` are measurable equivalences, but this should be true for any map, right?

Reply:
Right. I got a bit lazy here. It's fixed in the new version (and the proof is actually shorter). Thanks!

 I also added a `volume` version of the result since it seems that many results in this file are stated both ways.